railties/test/generators/plugin_generator_test.rb

```diff
@@ -463,10 +496,9 @@ def test_creating_dummy_application_with_different_name
 
   def test_creating_dummy_without_tests_but_with_dummy_path
     run_generator [destination_root, "--dummy_path", "spec/dummy", "--skip-test"]
-    assert_file "spec/dummy"
-    assert_file "spec/dummy/config/application.rb"
-    assert_no_file "test"
-    assert_no_file "test/test_helper.rb"
+    assert_directory "spec/dummy"
+    assert_file "spec/dummy/config/application.rb", /#\s+require\s+["']rails\/test_unit\/railtie["']/
```

Comment:
It isn't checked in another place, we need to check it.
A generation of plugin `--skip-test` has the more complicated case than a generation of app.

if `rails plugin new name_plugin --skip-test` and `--dummy-path` is `test/dummy`(default value the same), it woun't create a dummy app.
(we have condition for creation dummy app: `options[:skip_test].blank? || options[:dummy_path] != "test/dummy"`),
so it isn't possible to test `# require "rails/test_unit/railtie"` for existence in `config/application.rb`,

but we can reach it by `rails plugin new name_plugin --skip-test --dummy-path=spec/dummy`